content/fr/articles/js/es2015/object-literals.md

```diff
@@ -0,0 +1,82 @@
+---
+date: "2015-12-01"
+title: "ES6, ES2015 : Object literals"
+tags:
+  - JavaScript
+  - ES6
+  - ES2015
+authors:
+  - lionelb
+---
+
+Là encore, cette évolution du langage amène un sucre syntaxique bien pratique
+lors de l'initialisation d'objet.
+
+## Nom des propriétées raccourcies
+
+Lorsqu'on crée un objet, il arrive fréquemment que lorsqu'on définisse une
+propriété depuis une variable, la clé porte le même nom que la variable.
+
+```
+var firstname = "Robert";
+var lastname = "Laiponje";
+
+return {
+  firstname: firstname,
+  lastname: lastname
+}
+```
+
+Avec ES2015, on va pouvoir l'écrire comme ça&nbsp;:
```

Comment:
Je me rappelai plus si t'avais in trucs ou pas dans le doute je les ai rajouté. Mais in outils pour gérer la micro typo, ça serait cool effectivement.

> Le 7 déc. 2015 à 08:55, Maxime Thirouin <notifications@github.com> a écrit :
> 
> In content/fr/articles/js/es2015/object-literals.md:
> 
> > +## Nom des propriétées raccourcies
> > +
> > +Lorsqu'on crée un objet, il arrive fréquemment que lorsqu'on définisse une
> > +propriété depuis une variable, la clé porte le même nom que la variable.
> > +
> > +```
> > +var firstname = "Robert";
> > +var lastname = "Laiponje";
> > +
> > +return {
> > +  firstname: firstname,
> > +  lastname: lastname
> > +}
> > +```
> > +
> > +Avec ES2015, on va pouvoir l'écrire comme ça&nbsp;:
> nbsp? On devrait plutôt utiliser un truc qui ajuste la typo automatiquement (mais c'est un détail)
> 
> —
> Reply to this email directly or view it on GitHub.
>